tests/test_flow/fakes.py

```diff
@@ -150,28 +134,18 @@ def _create_aip_job(self, body, parent):
             "bionic.aip.main",
         ]
 
-        self._jobs[body["jobId"]] = ThreadPoolExecutor(max_workers=1).submit(
-            self._run_aip_job, body
-        )
-
-        return Mock()
-
-    def _run_aip_job(self, body):
         input_uri = body["trainingInput"]["args"][3]
 
         # AIP execution does not have access to the disk cache of the local
         # bionic instance. In order to emulate disk filesystem isolation, we can
         # change location of disk cache for the AIP job.
         self._modify_disk_cache_location(input_uri)
 
-        # AIP execution does not send back logs to local bionic instance. Hence,
-        # all logs are filtered here.
-        with temporarily_filter_all_logs():
-            try:
-                run_aip(input_uri, self._gcs_fs)
-                return "SUCCEEDED", None
-            except Exception as e:
-                return "FAILED", str(e)
+        self._jobs[body["jobId"]] = ProcessPoolExecutor(max_workers=1).submit(
```

Comment:
I am trying to keep this simple. Also, executors cannot be stored in `FakeAipClient` as executors are not serializable.

Reply:
Just documenting what we all discussed in a hangout:

1. If you create the AIP client entity with a function (instead of using `builder.set`) and disable persistence, then it shouldn't need to be parallelizable.
2. Let's try to check whether creating the ProcessPoolExecutor is expensive or not. If so, we can look into caching it.